dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -760,11 +763,30 @@ public Limits setUserLimits(@ApiParam(hidden = true) @Auth User authUser,
     @Path("/registries/{gitRegistry}/organizations/{organization}")
     @Operation(operationId = "getUserOrganizationRepositories", description = "Get all of the repositories for an organization for a given git registry accessible to the logged in user.", security = @SecurityRequirement(name = "bearer"))
     @ApiOperation(value = "See OpenApi for details")
-    public Set<String> getUserOrganizationRepositories(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user", in = ParameterIn.HEADER) @Auth User authUser,
-                                            @Parameter(name = "gitRegistry", description = "Git registry", required = true, in = ParameterIn.PATH) @PathParam("gitRegistry") SourceControl gitRegistry,
-                                            @Parameter(name = "organization", description = "Git organization", required = true, in = ParameterIn.PATH) @PathParam("organization") String organization) {
+    public List<Repository> getUserOrganizationRepositories(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user", in = ParameterIn.HEADER) @Auth User authUser,
+                                                           @Parameter(name = "gitRegistry", description = "Git registry", required = true, in = ParameterIn.PATH) @PathParam("gitRegistry") SourceControl gitRegistry,
+                                                           @Parameter(name = "organization", description = "Git organization", required = true, in = ParameterIn.PATH) @PathParam("organization") String organization) {
         Map<String, String> repositoryUrlToName = getGitRepositoryMap(authUser, gitRegistry);
-        return repositoryUrlToName.values().stream().filter(repository -> repository.startsWith(organization + "/")).collect(Collectors.toSet());
+        return repositoryUrlToName.values().stream()
+                .filter(repository -> repository.startsWith(organization + "/"))
+                .map(repository -> new Repository(repository.split("/")[0], repository.split("/")[1], gitRegistry, workflowDAO.findByPath(gitRegistry + "/" + repository, false, BioWorkflow.class).isPresent(), canDeleteWorkflow(gitRegistry + "/" + repository)))
+                .sorted(Comparator.comparing(Repository::getRepositoryName))
+                .collect(Collectors.toList());
+    }
+
+    /**
+     * Check if a workflow can be deleted.
+     * For now this is simply if a workflow is a stub or not
+     * @param path full path to workflow
+     * @return can delete workflow
+     */
+    private boolean canDeleteWorkflow(String path) {
+        Optional<BioWorkflow> workflow = workflowDAO.findByPath(path, false, BioWorkflow.class);
+        if (workflow.isPresent()) {
+            return Objects.equals(workflow.get().getMode(), WorkflowMode.STUB);
+        } else {
```

Comment:
Nitpick: can cleanly  remove the else clause and just return false 
(this set of braces is redundant).